app/services/DateService.scala

```diff
@@ -46,10 +46,12 @@ class DateService @Inject()(implicit val frontendAppConfig: FrontendAppConfig) e
       case _ => currentDate.getYear + 1
     }
   }
+
 }
 
 trait DateServiceInterface {
   def getCurrentDate: LocalDate
 
   def getCurrentTaxYearEnd(currentDate: LocalDate): Int
+
```

Comment:
it is actually fine to have extra method you created originally, idea was just to redesign things a bit